packages/material-ui/src/log-in-form.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import TextField from 'material-ui/TextField';
+import RaisedButton from 'material-ui/RaisedButton';
+import { validate } from '@accounts/client';
+import Layout from './layout';
+import ErrorMessage from './error-message';
+
+class LogInForm extends Component {
+  state = {
+    form: {
+      user: '',
+      password: '',
+    },
+    errors: {},
+    formError: null,
+  };
+
+  handleSubmit = async (e) => {
+    e.preventDefault();
+    const { accountsClient } = this.context;
+    const { form } = this.state;
+    this.setState({ formError: null, errors: {} });
+    const errors = validate.logIn(accountsClient, form);
```

Comment:
Yes it should handle validation. All validation should be handled by `@accounts/client`